knowledge_base/serverless_job/README.md

```diff
@@ -0,0 +1,38 @@
+# Serverless job
+
+This example demonstrates how to define and use a serverless job in a Databricks Asset Bundle.
+
+## Prerequisites
+
+* Databricks CLI v0.218.0 or above
+
+## Usage
+
+Configure the Databricks CLI profile to use:
+
+```bash
+export DATABRICKS_CONFIG_PROFILE=<my profile>
+```
```

Comment:
Not sure we need this step?

Reply:
It is to configure a workspace without hardcoding it in the configuration.

Alternatively, `--profile` but it's repetitive.

Reply:
But maybe we can just assume a default profile for the sake of this example? I think that's what https://github.com/databricks/bundle-examples/tree/main/knowledge_base/python_wheel_poetry did? Customers would probably just copy-past essential pasts rather than use the whole template anyway

Reply:
That example does hardcode an invalid workspace, prompting the user to change it in the readme.

Tbh, I don't like either of these, and really the intent is to make it work out of the box if you want to replicate it.

I'll copy the approach from the other template for consistency and we can revisit when we have a different approach for default auth in general (which we should do at some point).